### PR TITLE
feat(Accounts): The deposits are now being recorded as well as transactions

### PR DESCRIPTION
**Pull request by dejanvasic85**
Screen Shot 2019-05-22 at 1 44 04 pm
Screen Shot 2019-05-22 at 1 44 11 pm

### CODE REVIEW

File: server/db/models/Agent.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = (sequelize, type) => {
     businessName: type.STRING,
     abn: type.STRING,
     phone: type.STRING,
+    accountId: type.STRING,
```
Comment (dejanvasic85) [inline server/db/models/Agent.js]: Hmm this should be integer actually

**Comment by FastNinja** [comment]
`paymentMethod` is null - would be good if it is `stripe`
`description` - I am not sure - is it presentation logic? i.e. we can store payment source . as `creditCard`

**Comment by dejanvasic85** [comment]
paymentMethod was just added so it will be stored as "Credit Card". This is for presentation purpose.
paymentReference is the stripe Charge ID.
description i'm not sure. Happy to remove it.

**Comment by dejanvasic85** [comment]
There's no other comments? I'll merge and we can fix concerns later.

**Pull request merged**